sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -437,67 +437,77 @@ public Table getTable(TableReference tableRef) throws InterruptedException, IOEx
         (int) TimeUnit.MINUTES.toMillis(5);
 
     /**
+     * {@inheritDoc}
+     *
      * Tries to create the BigQuery table.
      * If a table with the same name already exists in the dataset, the table
      * creation fails.  In such a case,
```

Comment:
My general feeling was silently ignore the potential error might cause bad things in the future. I will think about it more.